### PR TITLE
picks kube 1.17.1

### PR DESCRIPTION
**Pull request by p0lyn0mial**

### CODE REVIEW

File: vendor/github.com/openshift/api/config/v1/types_feature.go [hunk 1]
```diff
@@ -97,44 +97,99 @@ type FeatureGateEnabledDisabled struct {
...
+}
+
+var defaultFeatures = &FeatureGateEnabledDisabled{
+	Enabled: []string{
+		"RotateKubeletServerCertificate", // sig-pod, sjenning
+		"SupportPodPidsLimit",            // sig-pod, sjenning
+		"NodeDisruptionExclusion",        // sig-scheduling, ccoleman
+		"ServiceNodeExclusion",           // sig-scheduling, ccoleman
+		"SCTPSupport",                    // sig-network, ccallend
+		"IPv6DualStack",                  // sig-network, ccoleman
```
Comment (p0lyn0mial) [inline vendor/github.com/openshift/api/config/v1/types_feature.go]: this causes `ci/prow/unit` to fail

```
--- FAIL: TestRenderCommand (0.10s)
    render_test.go:295: "scenario 1 checks feature gates" reports incorrect config file, error: expected to get exactly 7 feature gates but found 6: expected=[RotateKubeletServerCertificate=true SupportPodPidsLimit=true NodeDisruptionExclusion=true ServiceNodeExclusion=true SCTPSupport=true IPv6DualStack=true LegacyNodeRoleBehavior=false] got=[RotateKubeletServerCertificate=true SupportPodPidsLimit=true NodeDisruptionExclusion=true ServiceNodeExclusion=true SCTPSupport=true LegacyNodeRoleBehavior=false]
```

Comment (p0lyn0mial) [reply vendor/github.com/openshift/api/config/v1/types_feature.go]: @sttts does it mean we have to enable that feature in `cluster-kube-apiserver-operator` ?

Comment (p0lyn0mial) [reply vendor/github.com/openshift/api/config/v1/types_feature.go]: it seems to be coming from https://github.com/openshift/cluster-kube-apiserver-operator/blob/88c1f9f76d0078f8dc5580d64ceed2852f5f36ed/bindata/bootkube/config/bootstrap-config-overrides.yaml#L59

Comment (sttts) [reply vendor/github.com/openshift/api/config/v1/types_feature.go]: yeah, just update that list

**Pull request merged**